sympy/polys/matrices/tests/test_domainscalar.py

```diff
@@ -0,0 +1,82 @@
+from sympy.testing.pytest import raises
+
+from sympy.polys import ZZ, QQ
+from sympy.polys.matrices.domainscalar import DomainScalar
+
+
+def test_DomainScalar_init():
+    A = DomainScalar(ZZ(1), ZZ)
+    assert A.element ==  ZZ(1)
+    assert A.domain == ZZ
+
+
+def test_DomainScalar_new():
+    A = DomainScalar(ZZ(1), ZZ)
+    B = A.new(ZZ(4), ZZ)
+    assert B == DomainScalar(ZZ(4), ZZ)
+
+
+def test_DomainScalar_repr():
+    A = DomainScalar(ZZ(1), ZZ)
+    assert A.__repr__() == '1'
```

Comment:
This should test `repr(A)` rather than `A.__repr__()`.

Reply:
Thanks for the information.
Will do so.